### PR TITLE
[Roblox] Update subdomains

### PR DESCRIPTION
**Pull request by mark-otaris**
- accountsettings.roblox.com supports HTTPS
- billing.roblox.com supports HTTPS
- developer.roblox.com supports HTTPS
- devforum.roblox.com supports HTTPS
- friends.roblox.com supports HTTPS
- games.roblox.com supports HTTPS
- groups.roblox.com supports HTTPS
- partners.roblox.com no longer exists
- points.roblox.com supports HTTPS
- wiki.roblox.com supports HTTPS

### CODE REVIEW

**Comment by J0WI** [comment]
Triggering Travis again.

**Comment by pipboy96** [comment]
I'm not entirely sure what's going on here. Most likely it's some kind of rate limit.

File: src/chrome/content/rules/Roblox.xml [hunk 1]
```diff
@@ -1,85 +1,91 @@
...
+<ruleset name="Roblox">
 	<target host="roblox.com" />
+	<target host="www.roblox.com" />
 	<target host="abuse.roblox.com" />
+	<target host="accountsettings.roblox.com" />
 	<target host="affiliates.roblox.com" />
 	<target host="api.roblox.com" />
-	<target host="*.api.roblox.com" />
+	<target host="clientsettings.api.roblox.com" />
+	<target host="ephemeralcounters.api.roblox.com" />
```
Comment (pipboy96) [inline src/chrome/content/rules/Roblox.xml]:
```
voice.api.roblox.com
groups.api.roblox.com
games.api.roblox.com
files.api.roblox.com
sms.api.roblox.com
billing.api.roblox.com
search.api.roblox.com
api.roblox.com
users.api.roblox.com
marketplace.api.roblox.com
discussions.api.roblox.com
versioncompatibility.api.roblox.com
```

File: src/chrome/content/rules/Roblox-whitelist.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="Roblox (whitelisted domains)">
```
Comment (ivysrono) [inline src/chrome/content/rules/Roblox-whitelist.xml]: @zoracon Is this another whitelist way?

Comment (pipboy96) [reply src/chrome/content/rules/Roblox-whitelist.xml]: These are not whitelisted, maybe they used to be. All of them actually work.

**Pull request merged**